inventory/any.cf

```diff
@@ -184,6 +265,16 @@ bundle agent cfe_autorun_inventory_proc
       scope => "namespace";
 
   reports:
+    DEBUG|DEBUG_cfe_autorun_inventory_proc::
```

Comment:
I don't know of another way to see variable values. Perhaps the new verbose output addresses this somehow. Either way being able to enable extra output is very useful for debugging and understanding what's going on.